src/components/signInForm/index.tsx

```diff
@@ -0,0 +1,121 @@
+import { useForm, SubmitHandler } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useTranslation } from 'react-i18next';
+import { Inputs } from './types';
+import { StyledForm, PasswordLink, SubmitBtn, ErrorMsg } from './styles';
+import { useState } from 'react';
+import EyeIcon from 'src/assets/icons/iconEye';
+import EyeCloseIcon from 'src/assets/icons/iconEyeClose';
+import {
+  useLoginMutation,
+  useRestorePassMutation,
+} from 'src/redux/authApiSlice';
+import { useAppDispatch } from 'src/redux/hooks';
+import { setToken } from 'src/redux/auth/authSlice';
+import { useNavigate } from 'react-router-dom';
+import { useSignInSchema } from './signInFormHook';
+import {
+  isErrorWithMessage,
+  isFetchBaseQueryError,
+} from '../../helpers/errorHandler';
+
+export const SignInForm = () => {
+  const verifyRoute: string = '/';
+  const passwordRoute: string = '/restore-password';
```

Comment:
move constant above the component, in other case they will reinitialise with each rerender

Reply:
corrected in here https://github.com/ZenBit-Tech/faq_team_fe/pull/5/commits/bf58e21075e7ea2cbbb782951c2912de63f79f65#diff-e8d970e863c84aacdbb71ce91fbc891b1cc898246e59a71a471b1edfae921b35